otherlibs/unix/unix.ml

```diff
@@ -917,13 +917,13 @@ let create_process_gen usepath cmd args optenv
     (if new_stderr = 2 then 2 else file_descr_not_standard new_stderr)
   |] in
   Fun.protect ~finally:close_after
-    (fun () -> spawn cmd args optenv usepath redirections)
+    (fun () -> spawn cmd args optenv true (* usepath *) redirections)
```

Comment:
if `spawn` is not exported, we should make `usepath` a labelled argument instead of using comments to alleviate the problems of non-labelled boolean arguments.

(Looking at the code, I see that `spawn` is an external. Can external have labels too?)

Reply:
Externals can have labeled and even optional arguments.  There are examples elsewhere in this `unix.ml` file.

This said, `spawn` is not exported, and is called  only twice in `unix.ml`, so it doesn't need to have a bulletproof interface.

Reply:
OK, I'm not adding a label as it seems a bit overkill in this case...